dynamiqs/core/_utils.py

```diff
@@ -67,10 +74,30 @@ def compute_vmap(
                     in_axes = jax.tree_util.tree_map(lambda _: None, leaves)
                     in_axes[n - 1 - i] = 0
                     in_axes = jax.tree_util.tree_unflatten(treedef, in_axes)
-                    f = jax.vmap(f, in_axes=in_axes, out_axes=0)
+                    f = jax.vmap(f, in_axes=in_axes, out_axes=out_axes)
```

Comment:
Is this correct? 🤔

Reply:
Yes. Before it was batching the result over all batchable output arrays. So even `result.times` was batched, which doesn't make sense. We can change `out_axes` if we want to batch over different outputs, but the syntax is correct.

Reply:
Right, so this fixes a notable bug 🙏